### PR TITLE
docs: Add Kotlin math term atan

### PR DESCRIPTION
**Pull request by gonmmarques**
### Description
A new term entry for Kotlin .atan) method.
Closes https://github.com/Codecademy/docs/issues/3516

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+The **`.atan()`** method in Kotlin's `math` class is used to calculate the arc tangent of a value. It returns the value of the angle in the range from -PI/2 to PI/2 radians.
+
+## Syntax
+
+```pseudo
+atan(x)
+```
+
+- x: One of the two numbers whose hypotenuse is to be calculated.
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:23]:
```suggestion
- `x`: The number for which the arc tangent is to be calculated.
```

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The `.atan()` method returns a value of type `double` or `float` depending also on the type of the input respectively. There is a special case of:
+
+- atan(NaN) is NaN
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:27]:
```suggestion
- `atan(NaN)` is NaN
```

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (SSwiniarski) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:28]:
```suggestion
```
Remove extra blank line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@gonmmarques I did a first review and left a few comments.

**Review by SSwiniarski — APPROVED** [review]
@gonmmarques looks good for a second review.

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.atan()`** method in Kotlin's `math` class is used to calculate the arc tangent of a value. It returns the value of the angle in the range from -PI/2 to PI/2 radians.
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:15]:
```suggestion
The **`atan()`** method in Kotlin's `math` class is used to calculate the arc tangent of a value. It returns the value of the angle in the range from *-PI/2* to *PI/2* radians.
```

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Syntax
+
+```pseudo
+atan(x)
+```
+
+- `x`: The number for which the arc tangent is to be calculated.
+
+The `.atan()` method returns a value of type `double` or `float` depending also on the type of the input respectively. There is a special case of:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:25]:
```suggestion
The `atan()` method returns a value of type `double` or `float` depending also on the type of the input respectively. If the argument passed is `NaN`, the method will return `NaN`.
```

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+- `atan(NaN)` is NaN
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:27]:
```suggestion
```

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+## Example
+
+This example shows how to use the `.atan()` method to calculate the arc tangent of `90.0`:
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:31]:
```suggestion
This example shows how to use the `atan()` method to calculate the arc tangent of `90.0`:
```

File: content/kotlin/concepts/math-methods/terms/atan/atan.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+```kotlin
+import kotlin.math.*
+
+fun main(){
+    println(arctan(90.0))
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/atan/atan.md:37]:
```suggestion
    println(atan(90.0))
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@gonmmarques I've added my comments. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by gonmmarques** [comment]
> @gonmmarques I've added my comments. Please update the entry, and tag me when you've finished making the changes. Thanks

Thank you for your feedback :)

**Review by caupolicandiaz — APPROVED** [review]
@gonmmarques @SSwiniarski LGTM

**Comment by gonmmarques** [comment]
> @gonmmarques @SSwiniarski LGTM

Any to-dos missing from my side?

**Pull request merged**